packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "./ContentContainer.styles";
+import Confirmation from "./Confirmation/Confirmation";
+import Item from "./Item/Item";
+
+const { alignTypes, oneOf, node, string, func } = PropTypes;
+
+const propTypes = {
+  /** Alignment of the dropdown menu */
+  align: oneOf(alignTypes),
+
+  /** Children should consist of <Dropdown.Item /> */
+  children: node.isRequired,
+
+  /** Optional custom classes */
+  className: string,
+
+  /** Render prop for rendering the trigger element that toggles the dropdown */
+  renderTrigger: func.isRequired,
+};
+
+const defaultProps = {
+  align: "bottom",
+  className: "",
+};
+
+const DropDownMenu = props => {
+  const { align, children, className } = props;
+  const [isOpen, setIsOpen] = React.useState();
+  const [isConfirming, setIsConfirming] = React.useState();
+  const [renderConfirmation, setRenderConfirmation] = React.useState();
+
+  const handleCloseMenu = () => {
+    setIsOpen(false);
+    setIsConfirming(false);
+    setRenderConfirmation(null);
+  };
+
+  const handleOpenMenu = () => {
+    setIsOpen(true);
+  };
```

Comment:
As the Paprika Popover is now handling opening and closing keyboard shortcuts for its content, as well as handling popover closing when clicking outside of the content area, I felt handling dedicated `opening` or `closing` actions would be beneficial over a `toggle` action.